run/test/specs/invite_a_friend_share.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { bothPlatformsIt } from '../../types/sessionIt';
+import { newUser } from './utils/create_account';
+import { closeApp, openAppOnPlatformSingleDevice, SupportedPlatformsType } from './utils/open_app';
+import { AccessibilityId, USERNAME } from '../../types/testing';
+import { PlusButton, SearchButton } from './locators/home';
+import { AccountIDField, CloseButton, InviteAFriendOption, ShareButton } from './locators/start_conversation';
+import { MessageInput } from './locators/conversation';
+import { EnableButton } from './locators/global';
+import { NoteToSelfOption } from './locators/global_search';
+
+// TODO assign risk assessment value once PR 16 is merged (assumption: high)
+bothPlatformsIt('Invite a friend', inviteAFriend)
+
+let retrieveElement;
+
+async function inviteAFriend(platform: SupportedPlatformsType) {
+    const { device } = await openAppOnPlatformSingleDevice(platform);
+    // This is a const so that the user.accountID can be used later on 
+    const user = await newUser(device, USERNAME.ALICE, platform);
+    // Hit the plus button
+    await device.clickOnElementAll(new PlusButton(device));
+    // Select Invite a Friend
+    await device.clickOnElementAll(new InviteAFriendOption(device));
+    // Check for presence of Account ID field
+    await device.waitForTextElementToBePresent(new AccountIDField(device));
+    // Tap Share
+    await device.clickOnElementAll(new ShareButton(device));
+    // retrieving the "Hey..." message is different for each platform
+    if (platform === 'ios') {
+        // tap Copy in the native UI
+        await device.clickOnByAccessibilityID('doc.on.doc');
```

Comment:
What element is `doc.on.doc`?

Reply:
The doc.on.doc was the copy icon in the native UI but we do not need that anymore (see above)